buidler.config.js

```diff
@@ -1,4 +1,14 @@
 module.exports = {
+  network: {
```

Comment:
This is the root buidler.config.js. At this level, buidler is just used to compile contracts and not actually used to interact with a chain, so I don't think that we should tinker with network options at this level. It should work without this change.